cli/Cargo.toml

```diff
@@ -22,7 +22,7 @@ geojson = "0.17"
 ws = { version = "0.9.1", features = ["ssl"] }
 serde = { version = "1.0", features = ["derive"] }
 codec = { version = "2.0.0", package = "parity-scale-codec", features = ["derive"] }
-sgx_crypto_helper = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
+sgx_crypto_helper = { branch = "v1.1.4-testing", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
```

Comment:
Wouldn't it give us better control if we specified the revision here (i.e. git commit hash) ? In case they update the branch regularly

Reply:
There no 1.1.4 revision yet, it's only available as a branch. The git commit hash is already specified in the Cargo.lock. Also - if they update we should follow suit, so I don't see the use case of pinning down a specific commit in the Cargo.toml.

Reply:
But we should definitiely not stay on the branch, agreed. I'll create an issue for it that will need to update as soon as they have merged to master.

Reply:
I meant it more from the updating perspective. Obviously we don't want to stay on that branch for longer than necessary. So my question in the end is, would you rather update all the commit hashes in the `Cargo.lock` or in the `Cargo.toml`? I'd say there are fewer places in the `Cargo.toml`s to replace ?

Reply:
In the Cargo.lock because these can be updated with a simple (*cough*) cargo update. Unlike for the Cargo.toml where you'd have to manually replace the hashes.

Ah, I didn't know rev can also pin commits - anyway, I'm against pinning a commit, because a commit can be simply removed by a force push. Ofc, a branch can also be deleted, but I hope they only do that after publishing an official version 1.1.4

Reply:
Alright, I trust your better judgement on this 👍